net/tstun/wrap.go

```diff
@@ -843,15 +843,15 @@ func (t *Wrapper) Unwrap() tun.Device {
 	return t.tdev
 }
 
-// StatisticsEnable enables per-connections packet counters.
-// StatisticsExtract must be called periodically to avoid unbounded memory use.
-func (t *Wrapper) StatisticsEnable(enable bool) {
+// SetStatisticsEnabled enables per-connections packet counters.
+// [ExtractStatistics] must be called periodically to avoid unbounded memory use.
```

Comment:
remove the brackets

Reply:
We don't want to use godoc's new ability to hotlink identifiers?

Reply:
there's an official syntax now? oh. I thought this was just the latest of a dozen personal styles people try to use. I've seen backticks, pipes, single quotes, double quotes, ... I've always just said remove all of them since nothing was the convention.

Where are the docs on this new style?

Reply:
Proposed in https://github.com/golang/proposal/blob/master/design/51082-godocfmt.md#new-syntax-for-documentation-links, and accepted in https://github.com/golang/go/issues/51082.

Reply:
Removed it for now. I found a bug in `pkgsite` anyways, where it wasn't handling scopes properly.

Reply:
Oh, right, I've tuned those bugs out. I thought they'd never get approved 😂